dynamiqs/smesolve/euler.py

```diff
@@ -0,0 +1,20 @@
+from torch import Tensor
+
+from ..solvers.ode.fixed_solver import FixedSolver
+from .sme_solver import SMESolver
+
+
+class SMEEuler(SMESolver, FixedSolver):
+    def forward(self, t: float, rho: Tensor) -> Tensor:
+        # rho: (b_H, b_rho, ntrajs, n, n) -> (b_H, b_rho, ntrajs, n, n)
+
+        # sample Wiener process
+        dw = self.sample_wiener(self.dt)
+
+        # update state
+        drho = self.dt * self.lindbladian(t, rho) + self.diff_backaction(rho, dw)
```

Comment:
For consistency I would invert the two arguments in `self.diff_backaction` and put `rho` last.